store_json.go

```diff
@@ -11,32 +11,43 @@ import (
 )
 
 const (
-	customRacesDir    = "custom_races"
-	championshipsDir  = "championships"
+	maxAuditEntries   = 1000
+	//private data
```

Comment:
this is a picky one, but I like consistency within a codebase. please could you leave a space after your comments? e.g.

```go
// private data
```

Reply:
Picky, but also a fair one. Fixed already: cd8b5f4d94e2bc699d3000ff8b064d71f9e0032e